pipelines/internal/commands/export/export.go

```diff
@@ -34,6 +34,7 @@ var (
 	filter      = flags.String("filter", "", "the export filter")
 	datasetName = flags.String("dataset", "", "the dataset to export to which must already exist")
 	tableName   = flags.String("table", "", "the table to export to")
+	old         = flags.Bool("old", false, "export operations that aren't newer than those already exported")
```

Comment:
--all? to export all operations?

Reply:
I considered that but I thought the naming was confusing with alongside filter.  For example, if you are exporting all failed operations but stop half way through then you need to use this flag to get the rest since we list them newest first.

Reply:
Fair... Maybe --update (default true)?  I just don't think 'old' really tells us much.  It's more like 'pick up where you left off'.

Reply:
Done